lib/rubocop/runner.rb

```diff
@@ -236,18 +239,18 @@ def save_in_cache(cache, offenses)
 
     def do_inspection_loop(file)
       processed_source = get_processed_source(file)
-      offenses = []
+      offenses_by_iteration = []
```

Comment:
You might want to explain a bit that variable, as it took me a while to figure out what exactly does it track.

Reply:
Added a variable explanation comment